ds3-autogen-api/src/main/java/com/spectralogic/ds3autogen/api/models/Ds3Element.java

```diff
@@ -35,6 +35,16 @@ public Ds3Element(
         this.ds3Annotations = ds3Annotations;
     }
 
+    public Ds3Element(
+            final String name,
+            final String type,
+            final String componentType) {
+        this.name = name;
```

Comment:
Generally with these kinds of constructors you'll end up calling the other constructor directory, passing in null for the ignored param.